commands/backups/capture.js

```diff
@@ -12,6 +12,22 @@ function * run (context, heroku) {
   const interval = Math.max(3, parseInt(flags['wait-interval'])) || 3
   const db = yield fetcher.addon(app, args.database)
 
+  let dbInfo = yield heroku.request({
+    host: host(db),
+    method: 'get',
+    path: `/client/v11/databases/${db.id}`
+  }).catch(err => {
+    if (err.statusCode !== 404) throw err
+    cli.warn(`${cli.color.addon(db.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
```

Comment:
It seems weird to warn that the DB is not yet provisioned but then allow you to try to take a backup anyway. Are there any cases in which that can succeed?

Reply:
I don't know, but that seems unlikely to ever succeed. Maybe just bail out at that point?

Reply:
👍